### DISCUSSION TITLE
Output channel 'Open files': Open new tab / new window

### DISCUSSION THREAD

**Discussion by Taitava** [Taitava/obsidian-shellcommands#290] [Ideas - done & released]
## Two new flags for [Output channel: Open files](https://publish.obsidian.md/shellcommands/Output+handling/Output+channel+-+Open+files)
 - `:new-tab`: Sets `newLeaf` to `"tab"`. This actually happened already with Obsidian 0.16.x and 1.0.x when `new-pane` is used, but #276 reverts this behavior to open a new _pane_ like it used to do in Obsidian `0.15.x`.
 - `:new-window`: Sets `newLeaf` to `"window"`. This would be completely new behaviour to SC.

**Comment by pcause** [comment]
what about also having an option to output to a specific file where the file name can be specified either a fixed name of a note or with variables, and an option to append or replace

**Reply by Taitava** [reply]
Yea, it's a good idea. It's actually mentioned in this [long, old list of output channel ideas](https://github.com/Taitava/obsidian-shellcommands/discussions/16#discussion-3576047):
> - Insert output to another note in background (without opening the destination note) + defined positions: 1- caret position , 2-top , 3-bottom (option to apply line break \n after/before 1 2 3) - Thanks @FelipeRearden !

It's basically waiting for me to develop customisable settings for output channels. Should not be a big deal actually. But we can continue this discussion in #16, if needed. 🙂
